### PR TITLE
Add translator for svt.se

### PR DESCRIPTION
**Pull request by sebastian-berlin-wmse**
This is the website for Sveriges nyheter, the Swedish public service TV.

### CODE REVIEW

File: SVT Nyheter.js [hunk 1]
```diff
@@ -0,0 +1,341 @@
...
+	"translatorID": "4c6d887e-341d-4edb-b651-ea702a8918d7",
+	"label": "SVT Nyheter",
+	"creator": "Sebastian Berlin",
+	"target": "^https?://www.svt.se/nyheter/",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcibv",
```
Comment (adam3smith) [inline SVT Nyheter.js]: `gcisbv`

File: SVT Nyheter.js [hunk 2]
```diff
@@ -0,0 +1,341 @@
...
+	***** END LICENSE BLOCK *****
+*/
+
+
+// attr()/text() v2
+function attr(docOrElem,selector,attr,index){var elem=index?docOrElem.querySelectorAll(selector).item(index):docOrElem.querySelector(selector);return elem?elem.getAttribute(attr):null;}function text(docOrElem,selector,index){var elem=index?docOrElem.querySelectorAll(selector).item(index):docOrElem.querySelector(selector);return elem?elem.textContent:null;}
+
+
+function detectWeb(doc, url) {
+	return "newspaperArticle";
```
Comment (adam3smith) [inline SVT Nyheter.js]: We'd really want multiples, too, if at all possible

Comment (sebastian-berlin-wmse) [reply SVT Nyheter.js]: There's no search function on the site, so I don't think you can get multiples.

**Comment by sebastian-berlin-wmse** [comment]
I've updated according to the comments here and in #1691. Let me know if there's anything else.

**Comment by sebastian-berlin-wmse** [comment]
Is there anything else that needs fixing for this?

**Comment by adam3smith** [comment]
I don't think so, I'm just getting through reviews slowly, sorry.

**Comment by adam3smith** [comment]
Thanks!

**Pull request merged**